pyproject.toml

```diff
@@ -1,3 +1,86 @@
+[project]
+name = "nle"
+description = "The NetHack Learning Environment (NLE): a reinforcement learning environment based on NetHack"
+readme = "README.md"
+authors = [{ name = "The NLE Dev Team" }]
+requires-python = ">=3.10"
+dynamic = ["version"]
+classifiers = [
+  "License :: OSI Approved :: Nethack General Public License",
+  "Development Status :: 5 - Production/Stable",
+  "Operating System :: POSIX :: Linux",
+  "Operating System :: MacOS",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
+  "Programming Language :: C",
+  "Programming Language :: C++",
+  "Topic :: Scientific/Engineering :: Artificial Intelligence",
+  "Topic :: Games/Entertainment",
+]
+dependencies = ["pybind11>=2.2", "numpy>=1.16", "gymnasium==1.2.0"]
+
+[project.license]
+file = "LICENSE"
+
+[project.urls]
+Homepage = "https://github.com/NetHack-LE/nle"
+
+[build-system]
+requires = ["scikit-build-core>=0.10", "pybind11>=2.2", "setuptools-scm"]
+build-backend = "scikit_build_core.build"
+
+[tool.scikit-build]
+cmake.build-type = "Release"
+minimum-version = "build-system.requires"
+wheel.license-files = []
+cmake.args = ["-DHACKDIR=nle/nethackdir"]
+
+[tool.scikit-build.metadata.version]
+provider = "scikit_build_core.metadata.setuptools_scm"
+
+[tool.setuptools_scm]
+# This section is necessary to activate setuptools-scm, but can be empty
+
+[project.scripts]
+nle-play = "nle.scripts.play:main"
+nle-ttyrec = "nle.scripts.ttyrec:main"
+nle-ttyplay = "nle.scripts.ttyplay:main"
+nle-ttyplay2 = "nle.scripts.ttyplay2:main"
+nle-read-tty = "nle.scripts.read_tty:main"
+
+[dependency-groups]
+dev = [
+  "pre-commit>=2.0.1",
+  "isort>=5.13.2",
+  "cmake_format>=0.6.10",
+  "memory-profiler>=0.60.0",
+  "pytest>=6.2.5",
+  "pytest-benchmark>=3.4.1",
+  "sphinx>=2.4.4",
+  "sphinx-rtd-theme>=0.4.3",
+  "setuptools>=69.5.1",
+  "ruff>=0.4.3",
+]
+
+[project.optional-dependencies]
+agent = ["torch>=1.3.1"]
+all = [
+  "pre-commit>=2.0.1",
+  "isort>=5.13.2",
+  "cmake_format>=0.6.10",
+  "memory-profiler>=0.60.0",
+  "pytest>=6.2.5",
+  "pytest-benchmark>=3.4.1",
+  "sphinx>=2.4.4",
+  "sphinx-rtd-theme>=0.4.3",
+  "setuptools>=69.5.1",
+  "ruff>=0.4.3",
+  "torch>=1.3.1",
```

Comment:
This should be something like `{ include-group = "agent" }`

Reply:
Nice, thanks. This should now be fixed.

Reply:
Also fixed in #63